pvanalytics/metrics.py

```diff
@@ -4,6 +4,15 @@
 from pvlib.pvsystem import pvwatts_dc
 
 
+def _performance_ratio(measured, modeled):
+    return measured.sum() / modeled.sum()
```

Comment:
I think a docstring might be useful if this is meant to be generally useful for other formulations.